src/Common/CoreApi/Endpoint/Form/Form.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
this file could be _easily_ unit tested... 😇

Reply:
[651f16b](https://github.com/ems-project/EMSCommonBundle/pull/412/commits/651f16bb5e63802ed6b7d2a57ae69fc68fab08d2)